utils/users/src/insert.ts

```diff
@@ -0,0 +1,175 @@
+import _keycloakData from "./data_temp/keycloakData.json"
+import { KeycloakService } from "./database/keycloak"
+import axios from "axios"
+require("dotenv").config()
+interface IUser {
+  SECURE_PERSON_ID: string
+  FIRST_NAME: string
+  LAST_NAME: string
+  EMAIL_ADDRESS: string
+  keycloakData?: {
+    username: string
+    firstName: string
+    lastName: string
+    email: string
+    attributes: {
+      idir_user_guid: string[]
+      idir_username: string[]
+      displayName: string[]
+    }
+  }
+}
+
+interface IKeycloakData {
+  userData: IUser[]
+}
+
+enum SYSTEM_IDENTITY_SOURCE {
+  DATABASE = "DATABASE",
+  IDIR = "IDIR",
+  BCEID_BASIC = "BCEIDBASIC",
+  BCEID_BUSINESS = "BCEIDBUSINESS",
+  UNVERIFIED = "UNVERIFIED",
+}
+
+enum SYSTEM_USER_ROLE_NAME {
+  SYSTEM_ADMINISTRATOR = "System Administrator",
+  CREATOR = "Creator",
+  DATA_ADMINISTRATOR = "Data Administrator",
+}
+
+//SIMS USER SEED INTERFACE
+interface SystemUserSeed {
+  userIdentifier: string
+  identitySource: SYSTEM_IDENTITY_SOURCE
+  role_name: SYSTEM_USER_ROLE_NAME
+  userGuid: string
+  displayName: string
+  given_name: string
+  family_name: string
+  email: string
+}
+
+//get backbone api host
+const getBackboneApiHost = () =>
```

Comment:
This coalesce will never take the second branch, because if `process.env.API_HOST` and `process.env.API_PORT` are both null, the resulting string is `':'`, which is truthy